src/XMLGenerator.php

```diff
@@ -1247,8 +1247,8 @@ public function getLearnerReport(
         if (!empty($query->getUserStatus())) {
             $users->addChild('UserStatus', $query->getUserStatus());
         } else if (
-            !empty($query->getUserEmailAddresses()
-            || !empty($query->getUserEmployeeIds()))
+            !empty($query->getUserEmailAddresses())
+            || !empty($query->getUserEmployeeIds())
```

Comment:
This is a good change because it fixes an issues in the existing code.  But something about the structure

```pseudocode
function getLearnerReport() {
    // ...
    if () {
        foreach() {
            // ...
            if () {
             // ...
            } else if () {
                // ...
            } else {
                // ...
            }
        }
    }
}
```

... the level of nesting feels like there's probably something that could be cleaned up and made clearer.  But, apparently the new hotness in *LinkedIn and YouTube Dev Influencer Hot Takes* is to take shots at "Clean Code" as a concept worth pursuing, so what do I know? `¯\_(ツ)_/¯`